File: nivel-facil/solucao-java/FizzBuzz.java
public class FizzBuzz {
    public static void main(String[] args) {
        fizzBuzz(3);
        fizzBuzz(5);
        fizzBuzz(15);
        fizzBuzz(4);
    }

    public static void fizzBuzz(int number) {

        if(number % 3 == 0 && number % 5 == 0) {
            System.out.println("FizzBuzz");
        } else if(number % 3 == 0) {
            System.out.println("Fizz");
        } else if(number % 5 == 0) {
            System.out.println("Buzz");
        } else {
            System.out.println(number);
        }
    }
}

File: nivel-facil/solucao-kotlin/FizzBuzz.kt

fun main() {
    fizzBuzz(5)
    fizzBuzz(3)
    fizzBuzz(15)
    fizzBuzz(4)
}

fun fizzBuzz(number: Int): Unit {

    if(number % 3 == 0 && number % 5 == 0) {
        println("FizzBuzz")
    } else if(number % 3 == 0) {
        println("Fizz")
    } else if(number % 5 == 0) {
        println("Buzz")
    } else {
        println(number)
    }

}